### DISCUSSION TITLE
adding labels

### DISCUSSION THREAD

**Discussion by shaystings** [devcompl/TTController#130] [Q&A] [answered]
For starters I am not a coder. Currently my config is working I even got the speed to match ASUS rgb software speed, but i was wondering if there was a way to identify which fan is in which port with out having a separate list in a txt file. Most times this probably isn't an issue but I have 19 devices in my config if I want to move things around it takes a bit of extra time to first figure out witch is witch and than make the adjustments I want. I've done an example of the first 5 in bold. Do I need to add notation around the label? Is something like this possible?

{
  "Profiles": [
    {
      "Name": "Default",
      "Ports": [
      	[9802, 8101, 5]**back led**,
	[9802, 8102, 1]**exhast fan**,
      	[9802, 8104, 5]**top back**,
      	[9802, 8104, 4]**top mid**,
      	[9802, 8103, 3]**top front**,
      	[9802, 8104, 3],
      	[9802, 8104, 2],
      	[9802, 8104, 1],
      	[9802, 8102, 5],
      	[9802, 8102, 4],
      	[9802, 8102, 3],
      	[9802, 8102, 2]

**Comment by MoshiMoshi0** [comment, marked as answer]
You could do something like this:
```json
{
    "[9802, 8101, 5]": "back led",
    "[9802, 8102, 1]": "exhast fan",
    "[9802, 8104, 5]": "top back",
    "[9802, 8104, 4]": "top mid",
    "[9802, 8103, 3]": "top front",
    "Profiles": [
    {
        "Name": "Default",
        "Ports": [
```

These should be skipped when the config is loaded so they would work like comments/notes. 
But they have to have this format: `"<unique text>": "<any text>",` to avoid json errors. So you cant have 
```json
"a": "1", 
"a": "2"
``` 
if that makes sense.
I guess you can also use [Name](https://moshimoshi0.github.io/TTController/config/port#name) variable of PortConfigs to name single port or group of ports.
